spec/ics-2-consensus-requirements/README.md

```diff
@@ -0,0 +1,76 @@
+---
+ics: 2
+title: Consensus Requirements
+stage: proposal
+category: ibc-core
+author: Juwoon Yun <joon@tendermint.com>, Christopher Goes <cwgoes@tendermint.com>
+created: 2019-02-25
+modified: 2019-03-05
+---
+
+The IBC protocol creates a mechanism by which two replicated fault-tolerant state machines may pass messages to each other. These messages provide a base layer for the creation of communicating blockchain architecture that overcomes challenges in the scalability and extensibility of computing blockchain environments.
+
+The IBC protocol assumes that multiple applications are running on their own blockchain with their own state and own logic. Communication is achieved over an ordered message queue primitive, allowing the creation of complex inter-chain processes without trusted third parties.
```

Comment:
First two paragraphs seem like relatively common preamble that we can maybe eliminate?